osf/models/node.py

```diff
@@ -387,6 +388,8 @@ class AbstractNode(DirtyFieldsMixin, TypedModel, AddonModelMixin, IdentifierMixi
 
     schema_responses = GenericRelation('osf.SchemaResponse', related_query_name='nodes')
 
+    verified_resource_links = DateTimeAwareJSONField(null=True, blank=True)
```

Comment:
Renamed and updated options in my [PR](https://github.com/CenterForOpenScience/osf.io/pull/11099/files#diff-4b31ed2d6164722b2fcdd35dd56c11fa7cced0a4753944e56fc1807176593de3R398)

```python
addon_verified_resource_links = DateTimeAwareJSONField(default=dict, blank=True)
```